### ISSUE TITLE
ImageSharp WebP transparency issues

### ISSUE THREAD

**Issue by PerplexDaniel** [umbraco/Umbraco-CMS#16600]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

13.3.2

### Bug summary

There is a bug in ImageSharp that causes artifacts in transparent images when converted to WebP, see [this GitHub issue](https://github.com/SixLabors/ImageSharp/issues/2528). Is has already been fixed by the author in 3.1.4 (see [his comment](https://github.com/SixLabors/ImageSharp/issues/2528#issuecomment-1997301667) and [his fix](https://github.com/SixLabors/ImageSharp/pull/2699)).

Please update `SixLabors.ImageSharp` to 3.1.4 in all applicable Umbraco versions to fix the issue. 
I reproduced this on Umbraco 13.3.2 that uses SixLabors.ImageSharp 3.1.3.

For people who want to fix this issue now without waiting for Umbraco, you can install this newest version yourself for now:
`dotnet add package SixLabors.ImageSharp --version 3.1.4`

### Steps to reproduce

Use the latest Umbraco LTS (13.3.2) to reproduce.

<details>
<summary>bunny.png</summary>
<img src="https://github.com/umbraco/Umbraco-CMS/assets/24297947/12a8f077-06ae-4104-bb1c-c7a7e0c13dc5"/>
</details>

1. `dotnet new umbraco -n WebP`
2. `cd WebP`
3. `dotnet run`
4. Upload the attached image (bunny.png) to the media library
5. Go to the image URL (you can find it in the Media Library > Bunny.png > Info tab > Links)
6. Add `?format=webp` to the end
7. Observe the broken image, which looks like this (note all the weird black bars that should be transparent).

![image](https://github.com/umbraco/Umbraco-CMS/assets/24297947/cb4d94d6-ba5b-4adc-ab1d-d372b03f4954)

### Expected result / actual result

The WebP image should be fully transparent just like the original.

**Comment by Zeegaan** [comment]
This should be fixed in https://github.com/umbraco/Umbraco-CMS/pull/16616 😁

**Comment by Emetico** [comment]
I just updated from Umbraco CMS 13.3.2 to Umbraco CMS 13.4.1 and 13.4.1 uses Umbraco.Cms.Imaging.ImageSharp 13.4.1, but Umbraco.Cms.Imaging.ImageSharp 13.4.1 uses SixLabors.ImageSharp 3.1.3 and not 3.1.4.

**Comment by PerplexDaniel** [comment]
@Emetico Indeed, looking at NuGet it also says it uses `SixLabors.ImageSharp v3.1.3`:

![image](https://github.com/user-attachments/assets/67f894de-4009-4757-82dc-268b3f16e459)

And confirmed in Visual Studio:
![image](https://github.com/user-attachments/assets/a6cc71e0-804e-4df0-a22d-2b6c9545869d)

Which is confusing because the PR mentioned by @Zeegaan does bump it to `3.1.4`:

https://github.com/umbraco/Umbraco-CMS/pull/16616/files#diff-5baf5f9e448ad54ab25a091adee0da05d4d228481c9200518fcb1b53a65d4156R76

Perhaps it was reverted somewhere along the way?

**Comment by Zeegaan** [comment]
Heyo 👋 
Yea seems that PR has gotten overriden in a merge somehow... 💾  
We will make sure it goes in the next patch/minor, sorry about the confusion.. 😢

**Comment by DanDiplo** [comment]
Just a note that (in my experience) you need to delete the Media Cache to see the benefits of this.

**Comment by PerplexDaniel** [comment]
> Just a note that (in my experience) you need to delete the Media Cache to see the benefits of this.

Yes absolutely, that's a good thing to point out. ImageSharp will generate a crop with the same parameters only once and will cache it in `Umbraco/Data/TEMP/MediaCache` so if you have any broken crops in there they will be served until that folder is emptied.
